packages/spectral-test/package.json

```
@@ -25,28 +23,19 @@
     "clean": "rm -rf dist",
     "build": "yarn run format && yarn run lint && yarn run clean && tsc",
     "prepack": "yarn run build",
-    "format": "yarn run lint-fix && prettier --loglevel error --write 'src/**/*.ts' '*.{ts,js,json}'",
-    "check-format": "prettier --check 'src/**/*.ts' '*.{ts,js,json}'",
-    "check": "yarn run check-format && yarn run lint",
-    "lint": "eslint --ext .ts .",
-    "lint-fix": "eslint --fix --ext .ts .",
+    "lint": "biome lint .",
+    "lint-fix": "biome lint --apply-unsafe .",
+    "format": "yarn run lint-fix && biome format --write .",
```

```suggestion
    "format": "biome check --write",
```
Will run both lint and format.